python/engine/PythonEngine.cpp

```diff
@@ -20,18 +21,42 @@
 
 namespace openstudio {
 
-void PythonEngine::pyimport(const std::string& importName, const std::string& includePath) {
+void addToPythonPath(const openstudio::path& includePath) {
   if (!includePath.empty()) {
     PyObject* sys = PyImport_ImportModule("sys");
     PyObject* sysPath = PyObject_GetAttrString(sys, "path");
     // fmt::print("Prepending '{}' to sys.path\n", includePath);
-    PyObject* unicodeIncludePath = PyUnicode_FromString(includePath.c_str());
+    PyObject* unicodeIncludePath = PyUnicode_FromString(includePath.string().c_str());
     PyList_Insert(sysPath, 0, unicodeIncludePath);
   }
+}
+
+void PythonEngine::pyimport(const std::string& importName, const std::string& includePath) {
+  addToPythonPath(includePath);
   PyImport_ImportModule(importName.c_str());
 }
 
+void PythonEngine::setupPythonPath(const std::vector<openstudio::path>& includeDirs, const openstudio::path& pythonHomeDir) {
+  for (const auto& includeDir : includeDirs) {
+    addToPythonPath(includeDir);
+  }
+  if (!pythonHomeDir.empty()) {
+    wchar_t* a = Py_DecodeLocale(pythonHomeDir.generic_string().c_str(), nullptr);
+    Py_SetPythonHome(a);
+  }
+}
```

Comment:
So we can have cli args --python_path / --python_home (similar to gem_home / gem_path) to pick up python modules like numpy that are in a system python's standard lib.

eg this works:

```shell
Products/openstudio labs   --python_path /Users/julien/.pyenv/versions/3.8.12/lib/python3.8/lib-dynload   --python_path /Users/julien/.pyenv/versions/3.8.12/lib/python3.8/site-packages   -c "import numpy"
```